aap-common/proc-aap-list-available-playbooks.adoc

```diff
@@ -0,0 +1,77 @@
+[id="proc-aap-list-available-playbooks"]
+
+= Listing the available playbooks
+
+.Procedure
+. For the list of available playbooks without details, use the following command.
++
+[source,bash]
+----
+$ docker run —–-rm $IMAGE command_generator_vars | grep Playbook
+
+The current version of the operational playbooks collection contains the following playbooks:
+
```

Comment:
is is ok that aws is showing in gcp and gcp is showing in aws? Not sure whether you wanted to conditionalize the collection for each book or not?

Reply:
Yes it's a general list. I wondered about that, but the command produces both as an output.